docs/advanced_examples/DecisionTreeClassifier.ipynb

```diff
@@ -235,7 +235,8 @@
     }
    ],
    "source": [
-    "print(f\"Generating a key for an {circuit.graph.maximum_integer_bit_width()}-bit circuit\")"
+    "max_bit_width = circuit.graph.maximum_integer_bit_width(is_encrypted_filter=True)\n",
```

Comment:
what is `is_encrypted_filter=True` and shouldn't it be the default in CP? Looks like you've changed it in lot of places (or everywhere?)

Reply:
Everywhere in our code base.

I don't know but this is definitely a change in behavior from Concrete.

Reply:
maybe debate with @umut-sahin , because if you have to change, certainly the other users have to, and it's boring for them

Reply:
The default behavior didn't change, you can safely remove it as long as you don't use `show_assigned_bit_widths=True`, and this setting is a new feature so it's not breaking anything.